File: documentation/user/en/query/requirements/examples/telemetry/queryTelemetry.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		AttributeStartsWith("code", "garmin")
        	),
        	OrderBy(
        		AttributeNatural("code", Asc)
        	),
        	Require(
        		QueryTelemetry()
        	)
        )
	)
);

File: documentation/user/en/query/requirements/examples/telemetry/queryTelemetry.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					attributeStartsWith("code", "garmin")
				),
				orderBy(
					attributeNatural("code", ASC)
				),
				require(
					queryTelemetry()
				)
			)
		);
	}
);
